src/cmds/scripts/pbs_init.d.in

```diff
@@ -105,10 +105,10 @@ update_pids() {
 }
 
 # lc_host_name - convert host name into lower case short host name
-
+# also handle multiple names in PBS_LEAF_NAME
 lc_host_name() 
 {
-  echo $1 | cut -d: -f1 | cut -d. -f1 | sed -e "y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/"
+  echo $1 | cut -d, -f1 | cut -d: -f1 | cut -d. -f1 | sed -e "y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/"
```

Comment:
Please add a comment to state that PBS_LEAF_NAME is now of the format: **host:port,host:port** so the following code is to cut on , (to get the first host) and then : to parse our the port